readme.txt

```diff
@@ -249,6 +249,11 @@ You may set it to false to hide the public meeting finder altogether.
 
 To apply these changes, you must go to Settings > Permalinks and click "Save Changes"
 
+= Can I change the with_front configuration of the tsml meeting post type
```

Comment:
Could you change this to be the problem the user would be trying to solve by changing the configuration?

Reply:
I've updated the copy, let me know if that's ok.